images/runtime/dotnetcore/template.Dockerfile

```diff
@@ -23,4 +23,9 @@ ENV ORYX_AI_INSTRUMENTATION_KEY=${AI_KEY}
 COPY --from=startupCmdGen /opt/startupcmdgen/startupcmdgen /opt/startupcmdgen/startupcmdgen
 RUN ln -s /opt/startupcmdgen/startupcmdgen /usr/local/bin/oryx \
     && apt-get update \
-    && apt-get upgrade --assume-yes
\ No newline at end of file
+    && apt-get install unzip -y \ 
+    && apt-get upgrade --assume-yes
+
+RUN  mkdir -p /DotNetCoreAgent \
+     && curl -o /DotNetCoreAgent/appinsights.zip https://oryxsdks.blob.core.windows.net/appinsights-agent/DotNetCoreAgent.2.8.39.zip \
```

Comment:
We may want to remove the original zip file after uncompressing it into the DotnetCoreAgent folder

Reply:
good catch